media-libs/harfbuzz/harfbuzz_glib-8.2.1.recipe

```diff
@@ -0,0 +1,121 @@
+	2000-2005 Owen Taylor
+	2013, 2015-2017 Khaled Hosny
+	2010,2011,2012 Google, Inc.
+	2012 Mozilla Foundation
+	2011 Codethink Limited
+	2008,2010 Nokia Corporation and/or its subsidiary(-ies)
+	2009 Keith Stribley
+	2009 Martin Hosken and SIL International
+	2007 Chris Wilson
+	2005 David Turner
+	2004,2007,2008,2009,2010 Red Hat, Inc.
+	1998-2004 David Turner and Werner Lemberg"
+LICENSE="MIT"
+REVISION="1"
+SOURCE_URI="https://github.com/harfbuzz/harfbuzz/releases/download/$portVersion/harfbuzz-$portVersion.tar.xz"
+CHECKSUM_SHA256="0fec78f98c9c8faf228957a201c8846f809452c20f8445eb092a1ba6f22dbea5"
+SOURCE_DIR="harfbuzz-$portVersion"
+PATCHES="harfbuzz-$portVersion.patchset"
+
+ARCHITECTURES="all !x86_gcc2"
+SECONDARY_ARCHITECTURES="x86"
+
+commandSuffix=$secondaryArchSuffix
+commandBinDir=$binDir
+if [ "$targetArchitecture" = x86_gcc2 ]; then
+	commandSuffix=
+	commandBinDir=$prefix/bin
+fi
+
+libVersion="0.60821.0"
+libVersionCompat="$libVersion compat >= ${libVersion%%.*}"
+
+PROVIDES="
+	harfbuzz_glib$secondaryArchSuffix = $portVersion compat >= 0.9
+	lib:libharfbuzz_cairo$secondaryArchSuffix = $libVersionCompat
+	lib:libharfbuzz_gobject$secondaryArchSuffix = $libVersionCompat
+	"
+REQUIRES="
+	haiku$secondaryArchSuffix
+	harfbuzz$secondaryArchSuffix == $portVersion
+	lib:libcairo$secondaryArchSuffix
+	lib:libglib_2.0$secondaryArchSuffix
+	lib:libgobject_2.0$secondaryArchSuffix
+	"
+
+PROVIDES_devel="
+	harfbuzz_glib${secondaryArchSuffix}_devel = $portVersion compat >= 0.9
+	devel:libharfbuzz_cairo$secondaryArchSuffix = $libVersionCompat
+	devel:libharfbuzz_gobject$secondaryArchSuffix = $libVersionCompat
+	"
+REQUIRES_devel="
+	harfbuzz_glib$secondaryArchSuffix == $portVersion base
+	"
+
+BUILD_REQUIRES="
+	haiku${secondaryArchSuffix}_devel
+	devel:libcairo$secondaryArchSuffix
+	devel:libgirepository_1.0$secondaryArchSuffix
+	devel:libglib_2.0$secondaryArchSuffix
+	devel:libgobject_2.0$secondaryArchSuffix
+	"
+BUILD_PREREQUIRES="
+	cmd:gcc$secondaryArchSuffix
+	cmd:ld$secondaryArchSuffix
+	cmd:meson >= 0.55.0
+	cmd:ninja
+	cmd:pkg_config$secondaryArchSuffix
+	"
+
+BUILD()
+{
+	meson \
+		--buildtype=release \
+		--prefix=$prefix \
+		--libdir=$libDir \
+		--includedir=$includeDir \
+		-Dcairo=enabled \
+		-Dglib=enabled \
+		-Dgobject=enabled \
+		-Dintrospection=enabled \
+		-Dtests=disabled \
+		-Dutilities=disabled \
+		haiku_build
+	ninja -C haiku_build $jobArgs
+}
+
+INSTALL()
+{
+	ninja -C haiku_build install
+
+	# remove libtool library files
+	rm -f "$libDir"/libharfbuzz*.la
+
+	prepareInstalledDevelLibs \
+		libharfbuzz-cairo \
+		libharfbuzz-gobject
+	fixPkgconfig
+
+	# remove libraries provided in base harfbuzz package
+	rm -rf "$libDir"/libharfbuzz.*
+	rm -rf "$libDir"/libharfbuzz-subset.*
+	rm -rf "$developLibDir"/pkgconfig/harfbuzz.pc
+	rm -rf "$developLibDir"/pkgconfig/harfbuzz-subset.pc
+
+	# devel package
+	packageEntries devel \
+		"$dataDir" \
+		"$developDir" \
+		"$libDir"/cmake \
+		"$libDir"/girepository-1.0
```

Comment:
Not sure on this one, should the typelib files be kept in the base or devel package?

Reply:
Keeping it in base, no other package so far moves this to the devel package.